src/libexpr/flake/flake.cc

```diff
@@ -223,10 +228,41 @@ static Flake getFlake(
     } else
         throw Error("flake '%s' lacks attribute 'outputs'", lockedRef);
 
+    auto sNixConfig = state.symbols.create("nixConfig");
```

Comment:
I think we can safely rename this to only `config`.

Reply:
No, that would be ambiguous with `config` used in the NixOS module system.